src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -4487,6 +4494,8 @@ namespace Akka.Routing
         public virtual string RouterDispatcher { get; }
         public virtual bool StopRouterWhenAllRouteesRemoved { get; }
         public abstract Akka.Routing.Router CreateRouter(Akka.Actor.ActorSystem system);
+        [Akka.Annotations.InternalApiAttribute()]
+        public abstract Akka.Actor.ActorBase CreateRouterActor();
```

Comment:
Had to make this `public` for a Phobos-specific issue, namely trying to wrap Akka.NET's built-in routers for the purposes of telemetry injection.